pubsub/subscriptions/subscription_test.go

```diff
@@ -28,14 +31,14 @@ import (
 
 var topic *pubsub.Topic
 var subID string
-
+var client *pubsub.Client
```

Comment:
I think this and `setup` can be deleted?

Reply:
Modified to match topics_test